TestCases/pastix_support/readme.txt

```diff
@@ -65,6 +65,6 @@
 % - Ubuntu 18.04, gcc 7.4, ompi 3.1.4, mkl 2017, openblas 0.2.20 and 0.3.2.dev
```

Comment:
Well on the systems I have access to I get mpi runtime errors when I use icc (17 or 19) I don't think the bug is on my side...
I did test this on a variety of gcc / mpi / mkl / blas combinations so I think it is good to go.